DQM/BeamMonitor/plugins/OnlineBeamMonitor.cc

```diff
@@ -337,7 +409,7 @@ void OnlineBeamMonitor::fetchBeamSpotInformation(const Event& iEvent, const Even
       outFile << "BeamWidthY " << spotOnline.BeamWidthY() << std::endl;
       for (int i = 0; i < 6; ++i) {
```

Comment:
```suggestion
      for (int i = 0; i < reco::BeamSpot::dimension - 1; ++i) {
```


But why not
```suggestion
      for (int i = 0; i < reco::BeamSpot::dimension; ++i) {
```
also here?

Reply:
Ciao Andrea, thanks for spotting this! I think I just forgot it, I'll review it and fix it.

Reply:
Fixed in [73cfed7](https://github.com/cms-sw/cmssw/pull/47539/commits/73cfed7fc2de07607cb73770b8da0725bb9556ff)